src/include/sof/audio/module_adapter/intel_module_adapter/adsp_error_code.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
dirname should be `intel` instead of `intel_module_adapter` as it is already inside a directory named `module_adapter`.

Reply:
OK, will change it.

Reply:
Already fixed